smarts/sstudio/types.py

```diff
@@ -304,6 +304,17 @@ def __eq__(self, other):
         return self.__class__ == other.__class__ and hash(self) == hash(other)
 
 
+@dataclass
+class Via:
+    """A point on an edge that an actor must pass through"""
+
+    edge_id: str
+    lane_offset: int
+    lane_index: int
```

Comment:
lane_index is more closely related to edge_id
```suggestion
    lane_index: int
    lane_offset: int
```